main.go

```diff
@@ -24,7 +26,9 @@ const (
 	authPath                  = "auth"
 )
 
-type clusterSyncFn func(*client.APIClient) error
+// the 1st client represennts the pachd instance that needs to register with an enterprise server represented by the 2nd argument ("ec")
```

Comment:
minor nit spelling of represents

Reply:
is that not how you spell it? 😉

Reply:
it is! but there is a double nn in the code :)

Reply:
haha I think my sarcasm failed to come through :)

will update!

Reply:
lol! thanks :)